scripts/run_frontend_tests.py

```diff
@@ -29,6 +29,8 @@
 from . import common
 from . import install_third_party_libs
 
+DTSLINT_TYPE_TESTS = os.path.join('typings', 'tests')
```

Comment:
Can you make this a full path (use os.getcwd() or similar)?

Also please clarify in the naming whether this is a DIR_PATH or a FILE_PATH -- the name makes it sound like a list of tests

Reply:
I don't think I can include the full path here, the ```subprocess.Popen``` command is run from the cwd so ```DTSLINT_TYPE_TESTS_DIR_PATH``` needs to be relative:

```
cmd = ['./node_modules/dtslint/bin/index.js', DTSLINT_TYPE_TESTS_DIR_PATH]
task = subprocess.Popen(cmd, stdout=subprocess.PIPE)
```